db/migrate/20200131130342_create_chasers_sent.rb

```diff
@@ -0,0 +1,9 @@
+class CreateChasersSent < ActiveRecord::Migration[6.0]
+  def change
+    create_table :chasers_sents do |t|
+      t.string :type
+      t.integer :application_choice_id
```

Comment:
I think this needs to be a polymorphic association? So that it can be used for the both the reference chaser and RDB/DBD chasers.

Reply:
BTW foreign keys are `bigint` in Rails (see schema.rb). These columns might benefit from a `null: false` like other associations.

Reply:
okay cool, Just having a reading into polymorphic associations and will make the changes in a new migration 👍